tools/firmware-utils/src/tplink-safeloader.c

```diff
@@ -0,0 +1,3871 @@
+// SPDX-License-Identifier: BSD-2-Clause
```

Comment:
Please send a patch with your changes to the openwrt mailing list https://openwrt.org/submitting-patches

Reply:
firmware-utils was moved into a separate git repo here: https://git.openwrt.org/project/firmware-utils.git
I found it was easiest to check that out, make the commit locally, then use `git send-email` as described in *Sending patches by git send-email* above.
You can search the mailing list for "firmware-utils" examples of the commit messages people use
E.g. http://lists.openwrt.org/pipermail/openwrt-devel/2022-April/038421.html

Reply:
Ok, I am working on this separately. It should hopefully be approved soon.